src/extension.ts

```diff
@@ -83,14 +95,27 @@ export function activate(context: ExtensionContext) {
     let disposable = languageClient.start();
     toDispose.push(disposable);
     languageClient.onReady().then(() => {
+
+      // Code Lens actions
+      context.subscriptions.push(commands.registerCommand(Commands.SHOW_XML_REFERENCES, (uriString: string, position: Position) => {
```

Comment:
SHOW_REFERENCES, else it doesn't compile

Reply:
Sorry for this big mistake, @fbricon fixed.